releases/v1.2.8.toml

```diff
@@ -0,0 +1,37 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.7"
+
+pre_release = false
+
+preface = """\
+The eighth patch release for `containerd` 1.2 provides a series of bug fixes, many
+of them backported from the master branch to correct several known issues around
+manifest lists/indexes and pulling multi-arch, CVEs related to Golang/gRPC and libseccomp,
+fd leakage, and finally mount cleanup related to Cloud Foundry and rootless containers.
+
+### Notable Updates
+* Skip rootfs unmount when no mounts are provided. Fixed by [PR #3148](https://github.com/containerd/containerd/pull/3148) {cherry-picked as [PR #3402](https://github.com/containerd/containerd/pull/3402)}.
+* Close inherited socket file descriptor. Fixed in [PR #3359](https://github.com/containerd/containerd/pull/3359) {cherry-picked as [PR #3364](https://github.com/containerd/containerd/pull/3364)}.
+* Bump libseccomp-golang to v0.9.1. Fixed in [PR #3371](https://github.com/containerd/containerd/pull/3371) {cherry-picked as [PR #3376](https://github.com/containerd/containerd/pull/3376)}.
+* Call CloseIO when stdin closes in ctr. Fixed by [PR #3462](https://github.com/containerd/containerd/pull/3462) {cherry-picked as [PR 3490](https://github.com/containerd/containerd/pull/3490)}.
+* Several multi-arch image fixes, including: ARM platform matching, selecting the proper manifest, and limited to best matched manifest to solve discrepancies with multi-arch image operations. Backported [PR #3270](https://github.com/containerd/containerd/pull/3270) as [PR #3404](https://github.com/containerd/containerd/pull/3404), [PR #3484](https://github.com/containerd/containerd/pull/3484) as [PR #3512](https://github.com/containerd/containerd/pull/3512), and added [PR #3421](https://github.com/containerd/containerd/pull/3421).
+* Updated to build with the Golang 1.12.8 patch release to resolve CVE-2019-9512 and CVE-2019-9514. Originally fixed via [PR #3531](https://github.com/containerd/containerd/pull/3531) which lists the details of the Golang CVEs, backported via [PR #3532](https://github.com/containerd/containerd/pull/3532) to `release/1.2`.
```

Comment:
Follow-up for Go 1.12.9; https://github.com/containerd/containerd/pull/3544

(Relates to https://github.com/containerd/containerd/issues/3481, https://github.com/golang/go/issues/33405)